cmake/pcl_options.cmake

```diff
@@ -42,6 +42,10 @@ mark_as_advanced(PCL_NO_PRECOMPILE)
 option(PCL_ENABLE_SSE "Enable or Disable SSE optimizations." ON)
 mark_as_advanced(PCL_ENABLE_SSE)
 
+# Allow the user to disable ccache if ccache is available
+option(PCL_ENABLE_CCACHE "Enable using ccache (or clcache for MSVC) for compilation" OFF)
```

Comment:
Since we don't support MSVC/clcache, what about a) making option creation dependent on generator (i.e. do not create it at all for MSVC); b) removing "(or clcache for MSVC)" from the documentation?